src/trio/_subprocess.py

```diff
@@ -1164,7 +1163,7 @@ async def open_process(
         async def run_process(
             command: StrOrBytesPath,
             *,
-            stdin: bytes | bytearray | memoryview | int | HasFileno | None = None,
+            stdin: bytes | bytearray | memoryview | int | HasFileno | None = b"",
```

Comment:
Wondering why empty bytes instead of None here and below, what does this end up doing?

Reply:
In the docstring we say we default to this, it's just that I copied from typeshed which marks that the stdlib defaults to None.